HomeWork_One/Quiz_Template/src/components/Result_Components/ResultScreen.tsx

```diff
@@ -0,0 +1,41 @@
+import style from "./ResultScreen.module.css";
+import { Button } from "../buttons/Button.tsx";
+import { Statistic } from "./Statistic.tsx";
+import { FinishConfig } from "./FinishConfig.tsx";
+import { useDispatch, useSelector } from "react-redux";
+import { resetGameInfo } from "../../redux/gameInfo/gameInfoSlice.ts";
+import {
+  fetchQuestions,
+  resetQuestions,
+} from "../../redux/questions/questionsSlice.tsx";
+import { selectOption } from "../../redux/configuration/configSlice.ts";
+
+export function ResultScreen() {
+  const dispatch = useDispatch();
+  const config = useSelector(selectOption);
+
+  const handleGetQuestions = () => {
+    dispatch(resetGameInfo());
+    dispatch(
+      fetchQuestions(
```

Comment:
please check comment above
+ It is a bad practice to keep API calls inside components